README.md

```diff
@@ -22,6 +22,18 @@ EGO_CLIENT_ID=my-ego-application
 EGO_CLIENT_SECRET=secretpassword123
 ```
 
+## Vault
+
+Vault can be used to store your Ego credentials in a secure manner. When Vault is enabled, you will need to provide these env variables:
+```
+VAULT_URL=https://vault.my-vault-instance.com
+VAULT_ROLE=my-policy-name
+VAULT_SECRETS_PATH=/secrets-dir/path-to-secret/my-secret-name
+```
+Your Ego credential keys stored in your secret should match those listed in Ego Authorization.
```

Comment:
This isn't clear to me, but I think you are trying to say:
"The keys of the secrets stored in this vault path need to match the Ego environment variable names: `EGO_URL`, `EGO_CLIENT_ID`, and `EGO_CLIENT_SECRET`

Reply:
I should be even more specific since it's just the app credentials that need to go into vault, ego url can go in the config